src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,5 +9,8 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_PROPERTY_ID = new Prefix("b/");
```

Comment:
Should property prefix be "p" instead? Then bidder id be prefixed "b".

Reply:
i put b/ for now since there is the prefix PREFIX_PHONE using it but it shouldnt be a problem